src/cli/backup/exchange.go

```diff
@@ -270,27 +272,67 @@ func createExchangeCmd(cmd *cobra.Command, args []string) error {
 
 	sel := exchangeBackupCreateSelectors(user, exchangeData)
 
-	bo, err := r.NewBackup(ctx, sel)
+	// TODO(ashmrtn): Only call in the future if "*" or some other wildcard was
+	// given for resource owners.
+	users, err := m365.UserIDs(ctx, acct)
```

Comment:
I think the idea is that eventually we wouldn't fetch the users in GC during creation. Instead we'd just get an error when we discovered the user didn't exist? Getting users would only be required if a wildcard that needed resolved was given

Reply:
Fair.  Part of the "get all users" call is also used to verify that the graph configuration is successful/usable.  Always been a bit of a hack, so maybe this is a good time to look at separating the two cases.  Or, at least, separating when we choose to do each.

Reply:
I think this will help with that and make it easier to at least do "best effort" as each resource owner will now have an independent backup as well. For "fail fast" when we add it we'll need to consider how best to do the verification phase

Reply:
Fwiw, I'm thinking validation in terms of repo init/connect.  But I'm getting us into the weeds.  I agree with your assessment.